Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.9.1-slim-buster
 
 WORKDIR /app
 
-COPY . requirements.txt /app/
+COPY requirements.txt /app/
```

Comment:
@letsautom8, this changes your Dockerfile. Can you confirm this change is okay?

Reply:
Yes, it was intendent.

Reply:
thanks that was a type on my end. good catch @Ma007ks